lib/web/ui/server.go

```diff
@@ -152,6 +152,46 @@ func MakeKubeClusters(clusters []types.KubeCluster) []KubeCluster {
 	return uiKubeClusters
 }
 
+func MakeLegacyKubeClusters(servers []types.Server) []KubeCluster {
+	kubeClusters := map[string]*types.KubernetesCluster{}
```

Comment:
Suggestion:

```suggestion
	kubeClusters := map[string]*types.KubernetesCluster{} // name -> cluster
```

or call the variable `nameToCluster` ?